app/components/zoom-buttons.js

```diff
@@ -2,23 +2,19 @@ import Component from '@ember/component';
 
 import Zoom from 'ol/control/Zoom';
 
-export default Component.extend({
-  tagName: '',
+export default class extends Component {
+  tagName = '';
```

Comment:
AFAIK you should be using [`@tagName`](https://ember-decorators.github.io/ember-decorators/docs/api/modules/@ember-decorators/component#tagName) instead:

```ts
@tagName('')
export default class ZoomButtonsComponent extends Component {
  // ...
}
```

Reply:
yeah, but `tagName` is not officially supported and assigning it like that seems to work too 😅